modules/apps/headless/headless-object/headless-object-impl/rest-openapi.yaml

```diff
@@ -1,5 +1,24 @@
 components:
     schemas:
+        BatchEngineJobResponse:
+            properties:
+                batchId:
+                    description:
+                        The batch ID.
+                    format: int64
+                    type: integer
+                objectType:
```

Comment:
I think this name is confusing, as there are Objects in Liferay, so it seems it could be a type of Object.